rumqttc/src/lib.rs

```diff
@@ -304,7 +306,44 @@ impl From<ClientConfig> for TlsConfiguration {
 // TODO: Should all the options be exposed as public? Drawback
 // would be loosing the ability to panic when the user options
 // are wrong (e.g empty client id) or aggressive (keep alive time)
-/// Options to configure the behaviour of mqtt connection
+/// Options to configure the behaviour of MQTT connection
+/// 
+/// # Example: Use TLS transport
```

Comment:
Not sure if a full example is helpful here. It should be in the examples folder?